pom.xml

```diff
@@ -102,28 +101,16 @@
     <version.jakartaee-api>9.1-M2</version.jakartaee-api>
     <version.tomee-patch-plugin>0.9</version.tomee-patch-plugin>
 
-    <!--
-      JavaMail is both API and IMPL so we don't have them in the jakartaee-api uber jar.
-      We decided to add them here in the project so we can patch/update the 2 libraries without having
-      to update and release the jakartaee-api.jar. Different lifecycle.
-    -->
-    <geronimo-mail_2.1_spec.version>1.0.0-M1</geronimo-mail_2.1_spec.version>
-    <geronimo-mail_2.1_provider.version>1.0.0</geronimo-mail_2.1_provider.version>
```

Comment:
Any reason for dropping one of these versions? Guess we need to check, what is needed / required.

Reply:
relevant question, they are not dropped but moved, i redid the PR again and fixed the props names